charts/jindofs/templates/fuse/daemonset.yaml

```diff
@@ -108,6 +108,12 @@ spec:
                 fieldPath: status.hostIP
           - name: FLUID_FUSE_MOUNTPOINT
             value: {{ .Values.fuse.mountPath }}
+          - name: FLUID_RUNTIME_TYPE
```

Comment:
You also need to change jindofsx chart which is already used by default.

Reply:
Jindofsx is already changed here: f62964b